maat-court-data-api/src/test/java/gov/uk/courtdata/integrationTest/MockServicesConfig.java

```diff
@@ -34,4 +35,7 @@
     @MockBean(name = "cdaOAuth2WebClient")
     private WebClient cdaOAuth2WebClient;
 
+    @MockBean
+    private HearingResultedPublisher hearingResultedPublisher;
```

Comment:
We dont Mock the classes that we we worte. We mock third party Beans (eg: AmazonSQSConfig which actually needs the  mocking within HearingResultedPublisher )

Reply:
Well spotted and totally agreed.